### PR TITLE
Integrating policy checks into vrecord

### PR DESCRIPTION
**Pull request by aeschweik**
Supersedes #194, building on the work of @beccabender !

### CODE REVIEW

File: vrecord_policy_uncompressed.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            <policy type="and" name="Is it NTSC?">
+                <rule name="Is the standard NTSC?" value="Standard" tracktype="Video" operator="=">NTSC</rule>
+                <rule name="Is the height 486?" value="Height" tracktype="Video" operator="=">486</rule>
+                <rule name="Is the width 720?" value="Width" tracktype="Video" operator="=">720</rule>
+                <rule name="Is the frame rate 29.970?" value="FrameRate" tracktype="Video" operator="=">29.970</rule>
+                <rule name="Is the pixel aspect ratio 0.900?" value="PixelAspectRatio" tracktype="Video" operator="=">0.900</rule>
+                <rule name="Is the scan order bottom field first?" value="ScanOrder" tracktype="Video" operator="=">BFF</rule>
+                <rule name="Are the colour primaries BT.601 NTSC?" value="colour_primaries" tracktype="Video" operator="=">BT.601 NTSC</rule>
+                </policy>
+                <policy type="and" name="Is it PAL?">
```
Comment (retokromer) [inline vrecord_policy_uncompressed.xml]: I suggest to fix the alignment of this and the previous line, in order to visualise the structure.

**Comment by retokromer** [comment]
I guess, we should mention the policy checks in the documentation as well. Can you do the `README.md`? Then I will do the `man`, if something needs to be added there too. Thank you!

File: vrecord [hunk 2]
```diff
@@ -1281,3 +1281,27 @@ fi
...
+        if [[ "$STATUS" = "pass" ]] ; then
+            _report -dt "File passed policy check for uncompressed video."
+        else
+            report -wt "File did not pass vrecord policy check for FFV1 video and may not conform to digital preservation standards. Try another file?"
+            mediaconch -p "${SCRIPTDIR}/vrecord_policy_uncompressed.xml" "${dir}/${id}${suffix}.${extension}"
+            mediaconch -fx -p "${SCRIPTDIR}/vrecord_policy_uncompressed.xml" "${dir}/${id}${suffix}.${extension}" | xmlstarlet fo > "${dir}/${id}${suffix}_mediaconchreport.xml"
+            _report -wt "See ${dir}/${id}${suffix}_mediaconchreport.xml for a full MediaConch policy report."
+        fi
+fi
+if [[ "${video_codec_choice}" = "FFV1 version 3" ]] ; then
```
Comment (retokromer) [inline vrecord]: Is there any reason why you do not use `elif` here?

**Comment by dericed** [comment]
merging, thx @beccabender & @aeschweik !

**Comment by kieranjol** [comment]
Very cool, looking forward to trying it out!

**Pull request merged**